medical_medicament_us/models/medical_medicament_ndc.py

```diff
@@ -11,7 +11,7 @@ class MedicalMedicamentNdc(models.Model):
 
     name = fields.Char(
         string='NDC',
-        help='National Drug Code',
+        help='National Drug Code - 10-digit number',
```

Comment:
NDC can be 10 or 11 actually - http://phpa.dhmh.maryland.gov/OIDEOR/IMMUN/Shared%20Documents/Handout%203%20-%20NDC%20conversion%20to%2011%20digits.pdf